models/remoteworkitem.go

```diff
@@ -0,0 +1,67 @@
+package models
+
+import (
+	"encoding/json"
+)
+
+const (
+	SystemRemoteIssueId = "system.remote_issue_id"
```

Comment:
OK, in that case I think we should keep the name as it is ( "remote_issue_id" ) since it's being handled as a unique ID , just to keep things broad ?
Or do you suggest I update this to "remote_issue_url" ?